internal/policies/fetcher.go

```diff
@@ -19,20 +21,23 @@ import (
 
 // Fetcher fetches Kubewarden policies from the Kubernetes cluster, and filters policies that are auditable.
 type Fetcher struct {
+	// client knows about policies.kubewarden.io GVK
 	client client.Client
-	filter func(policies []policiesv1.Policy) []policiesv1.Policy
+	// list of skipped namespaces from audit, by name
+	skippedNs []string
+	filter    func(policies []policiesv1.Policy) []policiesv1.Policy
 }
 
 // NewFetcher returns a Fetcher. It will try to use in-cluster config, which will work just if audit-scanner is deployed
 // inside a Pod. If in-cluster fails, it will try to fetch the kube config from the home dir. It will return an error
 // if both attempts fail.
-func NewFetcher() (*Fetcher, error) {
+func NewFetcher(skippedNs []string) (*Fetcher, error) {
 	client, err := newClient()
 	if err != nil {
 		return nil, err
 	}
 
-	return &Fetcher{client: client, filter: filterAuditablePolicies}, nil
+	return &Fetcher{client: client, skippedNs: skippedNs, filter: filterAuditablePolicies}, nil
```

Comment:
(opening thread here, but this is about the following)

https://github.com/kubewarden/audit-scanner/blob/248d16d7c3e8dd843b37d6bf87374b91e9b722a5/internal/policies/fetcher.go#L43-L48

We don't need this function. I would like to think about the map-reduce again just in case this informs that we are tackling the problem incorrectly, but I think we can drop it.

Reply:
If we do not neeed it, drop it. :)

Reply:
Indeed, done in https://github.com/kubewarden/audit-scanner/pull/53/commits/40df6ef3b4223ab141f5c18a6447c295594ab227